roles/openshift_logging_kibana/tasks/generate_proxy_secret_files.yaml

```diff
@@ -0,0 +1,32 @@
+---
+# Check {{ generated_certs_dir }} for session_secret and oauth_secret
+- name: Checking for session_secret
+  stat:
+    path: "{{generated_certs_dir}}/session_secret"
+    get_checksum: false
+    get_attributes: false
+    get_mime: false
+  register: session_secret_file
+
+- name: Checking for oauth_secret
+  stat:
+    path: "{{generated_certs_dir}}/oauth_secret"
+    get_checksum: false
+    get_attributes: false
+    get_mime: false
+  register: oauth_secret_file
+
+# gen session_secret if necessary
+- name: Generate session secret
+  copy:
+    content: "{{ 32 | lib_utils_oo_random_word }}"
+    dest: "{{ generated_certs_dir }}/session_secret"
+  when: not session_secret_file.stat.exists or session_secret_file.stat.size > 50
```

Comment:
why 50?

Reply:
this was just copy and pasted to a new file so we could call it separate from the rest of the kibana install logic. its what we had before https://github.com/openshift/openshift-ansible/pull/11817/files#diff-eb8a7a231094de4642600a9bf48ce60fL87